src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -93,6 +96,10 @@ public function newAction(Request $request)
             // See http://symfony.com/doc/current/book/controller.html#flash-messages
             $this->addFlash('success', 'post.created_successfully');
 
+            if ($form['createAndShow']->isClicked()) {
```

Comment:
@yceruto, I think the `$form['createAndShow']` more short and clearly, let's wait for other people's opinions.